corehq/apps/app_manager/models.py

```diff
@@ -1124,9 +1124,9 @@ def _get_media_by_language(self, media_attr, lang, strict=False):
         media_dict = getattr(self, media_attr)
         if not media_dict:
             return None
-        if lang in media_dict:
+        if media_dict.get(lang, ''):
             return media_dict[lang]
```

Comment:
That's going to raise a KeyError if lang is not in media_dict. Don't you mean

    if strict:
        return media_dict.get(lang, '')
    else:
        # ...